keyboards/kinesis/kint36/kint36.h

```diff
@@ -0,0 +1,76 @@
+#pragma once
+
+#include "quantum.h"
+
+#define ___ KC_NO
+
+// This a shortcut to help you visually see your layout.
+// The first section contains all of the arguments as on the physical keyboard
+// The second converts the arguments into the 2-D scanned array
+
+#define LAYOUT(                                   \
+    kC0, kD0, kE0, kC1, kD1, kE1, kC2, kD2, kE2,  \
+    k00, k10, k20, k30, k40, k50,                 \
+    k01, k11, k21, k31, k41, k51,                 \
+    k02, k12, k22, k32, k42, k52,                 \
+    k03, k13, k23, k33, k43, k53,                 \
+         k14, k24, k34, k54,                      \
+                             k56, k55,            \
+                                  k35,            \
+                        k36, k46, k25,            \
+                                                  \
+    kC3, kD3, kE3, kC4, kD4, kE4, kC5, kE5, kD5,  \
+                   k60, k70, k80, k90, kA0, kB0,  \
+                   k61, k71, k81, k91, kA1, kB1,  \
+                   k62, k72, k82, k92, kA2, kB2,  \
+                   k63, k73, k83, k93, kA3, kB3,  \
+                        k64, k84, k94, kA4,       \
+              k96, k85,                           \
+              k86,                                \
+              k66, k75, k65                       \
+) {                                              \
+    { k00,  k01,  k02,  k03,  ___,  ___,  ___ }, \
+    { k10,  k11,  k12,  k13,  k14,  ___,  ___ }, \
+    { k20,  k21,  k22,  k23,  k24,  k25,  ___ }, \
+    { k30,  k31,  k32,  k33,  k34,  k35,  k36 }, \
+    { k40,  k41,  k42,  k43,  ___,  ___,  k46 }, \
+    { k50,  k51,  k52,  k53,  k54,  k55,  k56 }, \
+    { k60,  k61,  k62,  k63,  k64,  k65,  k66 }, \
+    { k70,  k71,  k72,  k73,  ___,  k75,  ___ }, \
+    { k80,  k81,  k82,  k83,  k84,  k85,  k86 }, \
+    { k90,  k91,  k92,  k93,  k94,  ___,  k96 }, \
+    { kA0,  kA1,  kA2,  kA3,  kA4,  ___,  ___ }, \
+    { kB0,  kB1,  kB2,  kB3,  ___,  ___,  ___ }, \
+    { kC0,  kC1,  kC2,  kC3,  kC4,  kC5,  ___ }, \
+    { kD0,  kD1,  kD2,  kD3,  kD4,  kD5,  ___ }, \
+    { kE0,  kE1,  kE2,  kE3,  kE4,  kE5,  ___ }, \
+}
+
+/*  ---------------- LEFT HAND -----------------   ---------------- RIGHT HAND ---------------- */
+#define LAYOUT_pretty(                                                                           \
```

Comment:
```suggestion
#define LAYOUT(                                   \
```

Reply:
I can’t seem to commit your suggestion? This is what I see:

![suggestion](https://user-images.githubusercontent.com/55506/91631922-954b3680-e9dd-11ea-8df6-b7824e50fab9.jpg)

Is there a mistake somewhere, or could you spell out what you’d like to see changed please?

Reply:
Personally, I'm thinking this change shouldn't be made, because it maintains compatibility with existing keymaps that may exist for this keyboard. Perhaps a user who already has one of the older controllers may want or need to upgrade later.

Reply:
Sorry, I don’t follow. What change are you against, and what compatibility are you concerned with? The kint36 variant I’m introducing here isn’t compatible with the old stapelberg controllers to begin with, so I don’t think that plays a role here?